[0.076s] Invoking command in '/home/ros2/exam/build/demo_interfaces_base': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake --build /home/ros2/exam/build/demo_interfaces_base -- -j12 -l12
[0.119s] -- Found ament_cmake: 1.3.12 (/opt/ros/humble/share/ament_cmake/cmake)
[0.261s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.263s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.267s] -- Found rosidl_adapter: 3.1.7 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.292s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.800s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.800s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.800s]   demo_interfaces_base --arguments-file
[0.800s]   /home/ros2/exam/build/demo_interfaces_base/rosidl_adapter__arguments__demo_interfaces_base.json
[0.800s]   --output-dir
[0.801s]   /home/ros2/exam/build/demo_interfaces_base/rosidl_adapter/demo_interfaces_base
[0.801s]   --output-file
[0.801s]   /home/ros2/exam/build/demo_interfaces_base/rosidl_adapter/demo_interfaces_base.idls)
[0.801s]   returned error code 1:
[0.801s] 
[0.801s]   Traceback (most recent call last):
[0.801s] 
[0.801s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.801s]       return _run_code(code, main_globals, None,
[0.801s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.801s]       exec(code, run_globals)
[0.801s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.801s]       sys.exit(main())
[0.801s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.802s]       abs_idl_file = convert_to_idl(
[0.802s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[0.802s]       return convert_srv_to_idl(
[0.802s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[0.802s]       srv = parse_service_string(package_name, input_file.stem, content)
[0.802s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 859, in parse_service_string
[0.802s]       request_message = parse_message_string(
[0.802s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 544, in parse_message_string
[0.802s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[0.802s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 388, in __init__
[0.802s]       self.base_type = BaseType(
[0.802s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.802s]       raise InvalidResourceName(
[0.802s] 
[0.802s]   rosidl_adapter.parser.InvalidResourceName: 'calculation_Request' is an
[0.802s]   invalid message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.802s] 
[0.803s] Call Stack (most recent call first):
[0.803s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.803s]   CMakeLists.txt:14 (rosidl_generate_interfaces)
[0.803s] 
[0.803s] [0m
[0.803s] -- Configuring incomplete, errors occurred!
[0.803s] See also "/home/ros2/exam/build/demo_interfaces_base/CMakeFiles/CMakeOutput.log".
[0.807s] gmake: *** [Makefile:747: cmake_check_build_system] Error 1
[0.812s] Invoked command in '/home/ros2/exam/build/demo_interfaces_base' returned '2': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake --build /home/ros2/exam/build/demo_interfaces_base -- -j12 -l12
